utils/repository.py

```diff
@@ -0,0 +1,11 @@
+def convert_list_to_tuple_string(id_list):
+    validate_list(id_list)
```

Comment:
validate_list(id_list): Does this get a list of ids? but the output of that function is completely ignored.
validate_list: should validate the list but it returns something

id_list[0]: I think you can use pop to get the firs value or use a map to reduce the list.
id_list rename for ids_list

I think this can be improved:
f"('{ids_list[0]}')" if len(ids_list) == 1 else str(tuple(ids_list))
for
[str(id) for id in ids_list]

or if you want tuples
(str(id) for id in ids_list)

Reply:
validate_list doesn't return a list, it only does assertions to validate if the id list is valid or not  -> if it is a list and if it has elements, because of that the output is ignored.
What I need is not the tuple, but the string of a tuple. For example '("1", "2")' instead of ("1", "2") 

When I only have one element and convert it to a tuple it would have something similar to ("1",) and this gives an error in the database because of the comma, for this reason,  maybe to make it better understood I could do something similar to this 
https://github.com/ioet/time-tracker-backend/blob/b095f56eea0509f56211724962449fbac3d53b8b/utils/extend_model.py#L69-L71

Reply:
I renamed id_list to ids_list in utils/repository.py 
https://github.com/ioet/time-tracker-backend/blob/79901cec1c2a39d2f5c4d49e6b6f426cc2bfb454/utils/repository.py#L1-L2

Also, I renamed in activity_model.py id_list to activities_ids_list 
https://github.com/ioet/time-tracker-backend/blob/79901cec1c2a39d2f5c4d49e6b6f426cc2bfb454/time_tracker_api/activities/activities_model.py#L66
And change the f"('{ids_list[0]}')" if len(ids_list) == 1 else str(tuple(ids_list)) to this
https://github.com/ioet/time-tracker-backend/blob/79901cec1c2a39d2f5c4d49e6b6f426cc2bfb454/utils/repository.py#L3-L6
I think with that changes it is more readability, the point of that is get a string of the tuple, without extra comma

Reply:
@scastillo-jp